packages/html-elements/package.json

```diff
@@ -38,15 +38,21 @@
     "projects": [
       {
         "preset": "jest-expo/ios",
-        "modulePathIgnorePatterns": ["<rootDir>/babel/"]
+        "modulePathIgnorePatterns": [
```

Comment:
See my comment from Doug's PR: https://github.com/expo/expo/pull/20913#discussion_r1083429167
I'm wondering why is this even trying to change this file 🤔

Reply:
hmm, I also see that on https://github.com/expo/expo/pull/20933, perhaps it has something to do with the `et uvm` command? Let me revert this

Reply:
fixed by rebasing this with `main`